.github/workflows/build-and-test.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: hhvm/actions/hack-lint-test@master
         with:
-          hhvm: nightly
+          hhvm: 4.168
```

Comment:
This effectively disables the job and HHAST auto-updating for nightlies - that might be fine (@Wilfred ? ), but if so, it's probably better to remove the job rather than change this

Reply:
Makes sense. It's currently broken on nightly, but for different reasons. Some other breaking change upstream causes it to fail https://github.com/hhvm/hhast/actions/runs/3991497523/jobs/6846355294

I don't think we can keep up with getting this to work for nightlies so might be ok to just go back to manual schema updates for the foreseeable future. Could remove or comment out this job.

Reply:
I removed the job and also reverted the docs PR about it back to the old instructions, which I followed to get the codegen working.